gap/Paths/paths.gd

```diff
@@ -1197,7 +1197,10 @@ DeclareAttribute( "ViewInformation", IsEdgeFacePath );
 #! @Section Waists
 #! @SectionLabel Waists
 #! This section deals with a specific type of closed vertex-edge-paths, namely waists.
-#! A n-waist is a closed vertex-edge path of length n such that all edges are inner and no two are incident to the same face.
+#! A <M>n</M>-waist is a closed vertex-edge path of length <M>n</M> such that all edges are inner and no two are incident to the same face.

```

Comment:
I tried to write the following paragraph based on the definition of a waist that is provided in the book. Maybe this can replace the lines 1200-1203.

In order to introduce the definition of a waist of a simplicial surface, we first present the definition of a distance-faithful path. Here, a closed edge path P of a given simplicial surface is called distance-faithful if for any two vertices of the path P at least one shortest edge path between them is contained in P. Using the above notion we introduce waists of simplicial surfaces.
Let therefore n be a natural number. If n satisfies 2 <=n<=3, then an n-waist of a given simplicial surface is defined as a circular edge path of length n such that all edges are inner and no two are incident to the same face.
Moreover, if the given simplicial surface is closed without 3-waists and the inequality n>=4 holds, we define an n-waist as a closed distance-faithful edge path of length n.